lesson07/script.js

```diff
@@ -1,14 +1,13 @@
-const loading = document.getElementById('js-loading');
-
 function createLoading(){
+    const loading = document.getElementById('js-loading');
     const loadingImg = document.createElement('img');
     loadingImg.src = "loading-circle.gif";
     loadingImg.alt = "ローディング画像";
     loading.appendChild(loadingImg);
 }
 
 function removeLoading(){
-    loading.remove();
+    document.getElementById('js-loading').remove();
 }
 
 function createList(value){
```

Comment:
I put the loading variable in the createLoading function.
Because the process is easier to understand when summarized.